tests/test_remove_python2_unicode_compatible.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests for verifying the output of the script edx_repo_tools/codemods/django3/remove_python2_unicode_compatible.py
+"""
+import os
+from unittest import TestCase
+import shutil
+import uuid
+import subprocess
+
+
+class Python2UnicodeCompatibleTests(TestCase):
+    """
+    Tests for the Python2_unicode_compatible.py output
+    """
+    def setup_local_copy(self, file_name):
+        """
```

Comment:
we should instead use `pytest`'s `tmpdir` , as used here https://github.com/edx/repo-tools/blob/1594e55dd4439d8860d203c478cd62008df0de8c/tests/test_modernize_openedx_yaml.py#L9

Reply:
Updated the function as per suggestion.